README.md

```diff
@@ -542,6 +542,40 @@ showCard(Pair('X', '♠'));
 // ! TypeError: ‘showCard’ expected a value of type (Pair Rank Suit) as its first argument; received Pair("X", "♠")
 ```
 
+#### `EnumType`
+
+`EnumType` is used to construct [enumerated types][5].
+
+To define an enumerated type one must provide:
+
+  - an array of values with distinct [`R.toString`][6] representations.
+
+```haskell
+EnumType :: [Any] -> Type
+```
+
+For example:
+
+```javascript
+//    TimeUnit :: Type
+const TimeUnit = $.EnumType(['milliseconds', 'seconds']);
+
+const def = $.create($.env.concat([TimeUnit, $.ValidDate, $.ValidNumber]));
+
+//    convertTo :: TimeUnit -> ValidDate -> ValidNumber
+const convertTo =
+def('convertTo',
+    {},
+    [TimeUnit, $.ValidDate, $.ValidNumber],
+    (unit, date) => date.valueOf() * (unit === 'milliseconds' ? 1000 : 1));
```

Comment:
Of course, I don't know why I thought that function returned seconds rather than milliseconds.